src/main/java/com/gymer/api/address/AddressRepository.java

```diff
@@ -1,13 +1,15 @@
 package com.gymer.api.address;
 
 import com.gymer.api.address.entity.Address;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.domain.Sort;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 interface AddressRepository extends PagingAndSortingRepository<Address, Long> {
 
-    Iterable<Address> findAllByCityContainsOrStreetContainsOrZipCodeContains(String city, String street, String zipCode, Sort sort);
+    Page<Address> findAllByCityContainsOrStreetContainsOrZipCodeContains(String city, String street, String zipCode, Pageable pageable);
```

Comment:
I suggest change name to findAllByCityOrStreetOrZipCodeContains. Little bit shorter and better to read in my opinion.